version/version.go

```diff
@@ -1,5 +1,7 @@
 package version
 
 var (
-	Version = "0.13.0"
+	Version = "0.14.0"
+	// PriorVersion - prior version
+	PriorVersion = "0.13.0"
```

Comment:
Is this being used anywhere? I didn't find any references in this PR.

Reply:
this is referred in csv-gen.go, for setting replaces tag for the previous CSV version.